src/lib/plugins/inventory.js

```diff
@@ -6,7 +6,7 @@ module.exports.player = function (player, serv, { version }) {
 
   player.heldItemSlot = 0
   player.heldItem = new Item(256, 1)
-  player.inventory = windows.createWindow(1, 'minecraft:inventory', 'inv', 36)
+  player.inventory = windows.createWindow(0, 'minecraft:inventory', 'inv', 36)
```

Comment:
can you explain this change?

Reply:
this is supposed to be 0 https://github.com/PrismarineJS/prismarine-windows/blob/7798a6858667c9cdf5d11d11a7101641c38b3cfb/index.js#L11